src/main/resources/application.yml

```diff
@@ -1,6 +1,6 @@
 spring:
   datasource:
-    url: jdbc:mysql://192.168.0.100:33061/mysql
+    url: jdbc:mysql://localhost:33061/mysql
```

Comment:
we should not use local host never, that was a recommendation by our trainer

Reply:
By default localhost points to the port in use at your local, this setting works fine on my computer.